extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -270,6 +271,13 @@ public void onConnectionLost(HueBridge bridge) {
     public void onConnectionResumed(HueBridge bridge) {
         logger.debug("Bridge connection resumed. Updating thing status to ONLINE.");
         updateStatus(ThingStatus.ONLINE);
+        // now also re-initialize all light handlers
+        for (Thing thing : getThing().getThings()) {
+            ThingHandler handler = thing.getHandler();
+            if (handler != null) {
+                handler.initialize();
```

Comment:
Again you are right, but this would not work for the hue binding, because it must be ensured, that the damned polling loop run at least once, before the light handler could initialize itself. This CANNOT be handled by the framework itself but only by the binding.